design/sketch.webidl

```diff
@@ -643,21 +642,21 @@ dictionary GPURenderPassColorAttachmentDescriptor {
     GPUTextureView attachment;
     GPUTextureView? resolveTarget;
 
-    GPULoadOp loadOp;
-    GPUStoreOp storeOp;
-    GPUColor clearColor;
+    GPULoadOp loadOp = "clear";
+    GPUStoreOp storeOp = "store";
+    GPUColor clearColor = {r: 0.0, g: 0.0, b: 0.0, a: 1.0};
 };
 
 dictionary GPURenderPassDepthStencilAttachmentDescriptor {
     GPUTextureView attachment;
 
-    GPULoadOp depthLoadOp;
-    GPUStoreOp depthStoreOp;
-    float clearDepth;
+    GPULoadOp depthLoadOp = "clear";
+    GPUStoreOp depthStoreOp = "store";
+    float clearDepth = 1.0;
```

Comment:
I don't find this obvious as well. In reverse-Z scenarios it would be cleared to zero.

Reply:
Since depth test is off by default, it seems ok to me to have a default clear depth. If a user enables depth testing, they can set this.

Reply:
It's unfortunate that we'd have a specific default there even for cases where we'd expect people to set it up...

Maybe we can turn this sideways completely by making the clear operation implicit:
```rust
float? clearDepth; // if NULL, don't clear
int? clearStencil;
```

Reply:
That seems like a pain for WebAssembly users. Just make me explicitly fill in the depth load op / depth clear value.

Reply:
That depends on the interface in wasm. In Rust, it would be great (better than JS): `enum DepthClear { Disabled, Enabled(f32) }`; in C, it doesn't need to be any more complicated that what the idl has now.

Reply:
nit: `required float? clearDepth; required int? clearStencil;`

Anyway, I kind of like this.